### PR TITLE
Do some minor cleanups before releasing a new beta

### PR DESCRIPTION
**Pull request by realityking**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,5 +2,3 @@ composer.phar
 composer.lock
 vendor/
 build/
-.idea/
```
Comment (Khaledgarbaya) [inline .gitignore:5]: switched editor ?

Comment (realityking) [reply .gitignore:5]: No. My CI tool told me I shouldn't put tool specific things into my project's .gitignore 😬

Comment (Khaledgarbaya) [inline .gitignore]: if you start sharing .idea in the github project it could cause some problems

**Review by Khaledgarbaya — APPROVED** [review]
LGTM 🚢

**Pull request merged**